lib/compat/wordpress-6.5/interactivity-api/class-wp-interactivity-api-directives-processor.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Interactivity API: WP_Interactivity_API_Directives_Processor class.
+ *
+ * @package WordPress
+ * @subpackage Interactivity API
+ */
+
+if ( class_exists( 'WP_Interactivity_API_Directives_Processor' ) ) {
+	return;
+}
+
+/**
+ * Class used to iterate over the tags of an HTML string and help process the
+ * directive attributes.
+ */
+class WP_Interactivity_API_Directives_Processor extends Gutenberg_HTML_Tag_Processor_6_5 {
```

Comment:
Rename `Gutenberg_HTML_Tag_Processor_6_5` with `WP_HTML_Tag_Processor` in WordPress.

@dmsnell can you confirm that this is correct?

Reply:
when we merge it into Core it needs to be WP_HTML_Tag_Processor` but while it remains in Gutenberg/as long as it remains in Gutenberg it should stay `Gutenberg_HTML_Tag_Processor_6_5` (or whatever version tracks Core `trunk`)

Reply:
Ok, thank you.